core/tests/bank_forks.rs

```diff
@@ -302,11 +340,12 @@ mod tests {
         // Purge all the outdated snapshots, including the ones needed to generate the package
         // currently sitting in the channel
         bank_forks.purge_old_snapshots();
-        let mut snapshot_paths = snapshot_utils::get_snapshot_paths(&snapshots_dir);
-        snapshot_paths.sort();
         assert_eq!(
```

Comment:
nits: (I merged this because `nits` doesn't worth to block this useful pr. Feel free to create mini follow-up pr if this bothers you enough.) you can write `assert_eq!(..., true);` with `assert!(...);`

Reply:
Fixed in #10721